commands/create.go

```diff
@@ -66,7 +66,7 @@ func runCreateCommand(cmd *cobra.Command, args []string) error {
 	}
 
 	// create project
-	if err := create.CreateProject(&projectCtx); err != nil {
```

Comment:
IMO `create.CreateProject` looks better than `create.Create`

Reply:
Decided to use `<command>.Run`.